Menus/MainMenu/MainMenu.cs

```diff
@@ -242,7 +238,7 @@ void CreateFaceButtons(List<Type> types)
 			}
 		}
 
-		void CreateFaceButton(MainMenuUI.ButtonData buttonData, ITooltip tooltip, Action buttonClickCallback)
+		void CreateFaceButton(MainMenuUI.ButtonData buttonData, ITooltip tooltip, Action buttonClickCallback, Type selectedType = null)
```

Comment:
My suggestion was a different approach--move the tool logic up into CreateFaceButtons